src/main/java/org/zalando/fahrschein/domain/Cursor.java

```diff
@@ -1,6 +1,6 @@
 package org.zalando.fahrschein.domain;
 
-public final class Cursor {
+public class Cursor {
```

Comment:
it's a general thing about not forcing tests code to create unnecessary instances.
Nevertheless, changed back to final, as it makes no sense to argue here about such a simple class.